vm-setup/roles/v1aX_integration_test/tasks/remediation.yml

```diff
@@ -52,7 +52,7 @@
         retries: 150
         delay: 3
         register: ready_nodes
-        until: ready_nodes.stdout == "2"
+        until: ready_nodes.stdout == NUMBER_OF_BMH
```

Comment:
if you run this without running the verify before, it will be undefined, no ? maybe better define it again at the beginning of this file

Reply:
Good catch!

Reply:
> This looks ok. it needs some basic documentation before going in, like a readme in the scripts/feature_tests folder and a note in the main readme to refer there maybe ?
Good point. We'll add it